packages/components/src/components/IconButton/index.tsx

```diff
@@ -29,7 +32,8 @@ export const IconButton: React.FC<IconButtonProps> = ({
         {
           width: '26px', // same width as (height of the button)
           paddingX: 0,
-          borderRadius: '50%',
+          borderRadius: variant === 'round' ? '50%' : '2px',
+          cursor: variant === 'round' ? 'pointer' : 'default',
```

Comment:
We could add a comment here explaining why `round` gets a pointer and `square` doesn't for future reference maybe.

Reply:
right!